src/shared/icons/LeftArrow.tsx

```diff
@@ -0,0 +1,3 @@
+export const LeftArrow = (): JSX.Element => <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+  <path d="M7.14899 11.1518L16.8802 2L18 3.16432L8.29832 12.2883L17.9817 21.8638L16.8327 23L6 12.288L7.14899 11.1518Z" fill="#263238"/>
```

Comment:
We can also use tailwind for styling svg. Please, use custom colors from tailwind.config.js.
Read more: https://tailwindcss.com/docs/fill#setting-the-fill-color

Reply:
It does not work and I use CSS.

Reply:
Look at this response: https://github.com/Vladyslav-Samokryk/gift-hub-frontend/pull/13#discussion_r1304804186